atom/browser/native_window_views.cc

```diff
@@ -874,6 +872,33 @@ void NativeWindowViews::HandleSizeEvent(WPARAM w_param, LPARAM l_param) {
 }
 #endif
 
+gfx::Size NativeWindowViews::WindowSizeToFramelessSize(
+    const gfx::Size& size) {
+  if (size.width() == 0 && size.height() == 0)
+    return size;
+
+  gfx::Rect window_bounds = gfx::Rect(size);
+  if (use_content_size_) {
+    if (menu_bar_ && menu_bar_visible_) {
+      window_bounds.set_height(window_bounds.height() + kMenuBarHeight);
+    }
+  } else if (has_frame()) {
+#if defined(OS_WIN)
+  gfx::Size frame_size = gfx::win::ScreenToDIPRect(
```

Comment:
The `WinFrameView::GetMinimumSize` calls `gfx::win::DIPToScreenSize` for the passed size, maybe we should just remove the call there? So we don't need to convert screen size to DPI size and then convert back.